tests/unit/test_reporter.py

```diff
@@ -0,0 +1,77 @@
+import pytest
```

Comment:
Wow, this will increase the coverage a lot! 😍😍😍

Reply:
Yes! We didn't have any tests for the reporter yet so it's increasing a good 7.31% ⬆️